jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -9,9 +9,12 @@ order: 15
 
 *[2.0]({{ site.baseurl }}/2.0/) > Migrating from 1.0 to 2.0*
 
-This document will give you a starting place for migrating from CircleCI 1.0 to 2.0 by using a copy of your existing 1.0 configuration file and replacing the old keys with the new keys if equivalents exist. The migration process may not end with this document, but the goal is to get the majority of keys replaced with the equivalent syntax nesting and to help you get started with adding new functionality.
+This document provides a starting point for migrating from CircleCI 1.0 to 2.0.
```

Comment:
We've had positive customer feedback on this introduction in the usability studies, so I'd like to keep it as-is. Otherwise, this looks great!